src/convert.jl

```diff
@@ -420,26 +420,27 @@ Convert from line segment to polygon in H-representation.
 
 ### Input
 
-- `type` -- target type
-- `L`    -- line segment
-
+- `type`  -- target type
+- `L`     -- line segment
+- `prune` -- (optional, default: `false`) flag for removing redundant
+             constraints in the end
 ### Output
 
 A flat polygon in constraint representation with the minimal number of
 constraints (four).
 """
-function convert(::Type{HPOLYGON}, L::LineSegment{N}
-                ) where {N<:Real, HPOLYGON<:AbstractHPolygon}
+function convert(::Type{HPOLYGON}, L::LineSegment{N};
+                 prune=false) where {N<:Real, HPOLYGON<:AbstractHPolygon}
```

Comment:
Adding the `prune` argument in the conversion of a `LineSegment` does not make sense because there are always two constraints and none of them is ever redundant.

Reply:
You are right, ill remove this argument.